lib/ansible/plugins/inventory/aws_ec2.py

```diff
@@ -17,13 +17,18 @@
     description:
         - Get inventory hosts from Amazon Web Services EC2.
         - Uses a YAML configuration file that ends with aws_ec2.(yml|yaml).
+    notes:
+        - If no credentials are provided and the control node has an associated IAM instance profile then the
+          role will be used for authentication.
     options:
         plugin:
             description: token that ensures this is a source file for the 'aws_ec2' plugin.
             required: True
             choices: ['aws_ec2']
         boto_profile:
-          description: The boto profile to use.
+          description: The boto profile to use. This plugin supports boto3-style credentials, so the profile
```

Comment:
I normally split this sort of statement onto multiple lines. Something similar to...

```
description:
- The boto profile to use.
- This plugin supports boto3-style...
- See U(...
```

Reply:
That's tidier, good idea. I updated the plugin so it would be consistent.